lib/board.rb

```diff
@@ -0,0 +1,151 @@
+# rubocop:disable Naming/MethodParameterName,Metrics/CyclomaticComplexity,Metrics/MethodLength,Metrics/PerceivedComplexity
+
+class Board
+  attr_reader :board, :current_player, :game_running, :player1, :player2, :current_move, :winner
+
+  def initialize(player1, player2)
+    @board = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ]
+    @played_moves = []
+    @current_player = player1
+    @is_won = false
+    @game_running = true
+    @winner = nil
+    @player1 = player1
+    @player2 = player2
+    @current_move = nil
+  end
+
+  def display_board
+    @board.each do |row|
+      puts
+      row.each do |block|
+        print block.to_s + ' '
+      end
+      puts
+    end
+    puts
```

Comment:
**_No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))_**